DonutsGo/DonutsGo.Application/Models/Products/UpdateProductModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DonutsGo.DataAccess.Entities;
+
+namespace DonutsGo.Application.Models.Products;
+
+public class UpdateProductModel
+{
+    public string Name { get; set; }
+
```

Comment:
Consistency is also a key point in coding. The properties from ProductResponseModel above have no empty lines between them but these here do. You need to choose one approach and use it everywhere.